101-bundle-json.md

```diff
@@ -852,13 +852,131 @@ The fields are defined as follows:
   - `EXTENSION NAME`: a unique name for an extension. Names SHOULD follow the dotted name format described earlier in this section.
   - The value of the extension must be valid JSON, but is otherwise undefined.
 
-The usage of extensions is undefined. However, bundles SHOULD be installable by runtimes that do not understand the extensions.
+### Required Extensions
+
+Some extensions MAY be required by a bundle and operations involving that bundle MUST fail if the runtime does not support that bundle. In this case, a bundle author MUST use the `requiredExtensions` array to define those extensions that are necessary for installing the bundle. The `requiredExtensions` array should contain the `EXTENSION NAME` defined in the `custom` object for any extensions that are required for bundle operations.
```

Comment:
I'm with pedantry: "...are necessary for performing all actions with the bundle...." or some such.